### ISSUE TITLE
[1] When Tabs component collapses to dropdown, subText disappears

### ISSUE THREAD

**Issue by sschiessl-bcp** [bitshares/bitshares-ui#1278]
This is the classes setup
```
<Tabs
                            ref={instance => { this.tabs = instance; }}
                            className="account-tabs"
                            tabsClass="account-overview bordered-header content-block"
                            contentClass="padding"
                            segmented={false}
                        >
```

Expanded view:
![image](https://user-images.githubusercontent.com/33128181/37150690-e7f40e22-22d2-11e8-8240-d6ae9fc0f7d8.png)

Collapsed view:
![image](https://user-images.githubusercontent.com/33128181/37150674-d93ac40c-22d2-11e8-8698-8949df0516e5.png)

**Comment by ahdigital** [comment]
I can take this one.

**Comment by sschiessl-bcp** [comment]
I already added a fix for it #1279 

Sorry for the confusion, I thought it was clear with the commit message above.

**Comment by ahdigital** [comment]
Looks good to me. (I deleted my last comment. I was on my phone at the time so didn't see the link to the PR.)
